src/Microsoft.OData.Core/ODataAnnotatable.cs

```diff
@@ -11,21 +11,21 @@
 
 namespace Microsoft.OData
 {
-    /// <summary>
-    /// Base class for all annotatable types in OData library.
-    /// </summary>
+	/// <summary>
+	/// Base class for all annotatable types in OData library.
+	/// </summary>

```

Comment:
Fix formatting. You could restore the file back then reapply your changes since they only affect the private field and the public property